### PR TITLE
i-padiolleau_proposal_late-attempt

### PR DESCRIPTION
**Pull request by i-padiolleau**

### CODE REVIEW

File: project/i-padiolleau/README.rst [hunk 1]
```diff
@@ -0,0 +1,49 @@
+##Basic Information
+https://github.com/i-padiolleau/optimization-for-composite-type-data-structure
+
+##Problem to Solve
+When it comes to optimize a code that manipulate at signicant amount of data, it is important to consider how the data is access to adapt the data structure used in the code.
```
Comment (yungyuc) [inline project/i-padiolleau/README.rst:5]: What will be the problem that manipulate significant amount of data in your project?

File: project/i-padiolleau/README.rst [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+**Git will** serve as **version control** : Each week will be marked as V+1.0 and modified features marked as VX.+1
+**Testing** will be perform using **python3.pytest**
+**Documentation** (comments and **README** file to explain the few steps to use the code)
+
+##Schedule
+Week 1 (10/31):
+Surrend proposal and worked the problem on paper, to some research online about data locality and tests demonstrating this principle
+
+Week 2 (11/7):
+Make the structure of the C++ code, Makefile
```
Comment (yungyuc) [inline project/i-padiolleau/README.rst:31]: Without specific problems, it is very difficult to come up with code or data structures.

**Review by yungyuc — CHANGES_REQUESTED** [review]
Do not mix changesets and files of a homework or unrelated things.  Please let me a message once you fix it.

Please add your project to the list in the readme file in the `project` root directory.

File: project/i-padiolleau/README.rst (hunk 1 above)
Comment (i-padiolleau) [reply project/i-padiolleau/README.rst:5]: I add specification on the problem I wnat to work on. Its about population data computation

File: project/i-padiolleau/README.rst (hunk 2 above)
Comment (i-padiolleau) [reply project/i-padiolleau/README.rst:34]: Its corrected

**Comment by i-padiolleau** [comment]
> 

I fixed it and added the project information in README file

File: project/i-padiolleau/README.rst (hunk 1 above)
Comment (yungyuc) [reply project/i-padiolleau/README.rst:5]: The tangible objective will help you focus.  It sounds good.

**Review by yungyuc — APPROVED** [review]

**Comment by yungyuc** [comment]
You did not fix the issue of changeset history, but I can fix it for you.  Please try to figure out what went wrong in the future.

**Pull request merged**